orangecontrib/text/widgets/owtwitter.py

```diff
@@ -84,19 +132,17 @@ class Outputs:
     resizing_enabled = False
 
     class Warning(OWWidget.Warning):
-        no_text_fields = Msg('Text features are inferred when none selected.')
+        no_text_fields = Msg("Text features are inferred when none selected.")
 
     class Error(OWWidget.Error):
-        api = Msg('Api error ({})')
-        rate_limit = Msg('Rate limit exceeded. Please try again later.')
-        empty_authors = Msg('Please provide some authors.')
-        wrong_authors = Msg('Query does not match Twitter user handle.')
-        key_missing = Msg('Please provide a valid API key to get the data.')
-
-    tweets_info = 'Tweets on output: {}'
+        api = Msg("Api error ({})")
```

Comment:
Oh, just noticed this. api should be renamed to api_error, else someone would assume this contains some api info.

Reply:
done